rules/azure/execution_consent_grant_attack_via_azure_registered_application.toml

```diff
@@ -43,15 +43,21 @@ event.dataset:(azure.activitylogs or azure.auditlogs or o365.audit) and
 [[rule.threat]]
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
-id = "T1192"
-name = "Spearphishing Link"
-reference = "https://attack.mitre.org/techniques/T1192/"
+id = "T1566"
+name = "Phishing"
+reference = "https://attack.mitre.org/techniques/T1566/"
 
+[[rule.threat.technique.subtechnique]]
+id = "T1566.002"
+name = "Spearphishing Link"
+reference = "https://attack.mitre.org/techniques/T1566/002/"
 
 [rule.threat.tactic]
-id = "TA0001"
-name = "Initial Access"
-reference = "https://attack.mitre.org/tactics/TA0001/"
```

Comment:
Spear phishing falls under initial access, I don't think this should've been updated